pandas/tests/frame/test_apply.py

```diff
@@ -250,6 +250,13 @@ def test_apply_raw(self, float_frame):
         expected = float_frame * 2
         tm.assert_frame_equal(result, expected)
 
+        # Mixed dtype
```

Comment:
can you move the _assert_raw to the top of this test & use it *also* on a single dtype frame (like you do below but on float_frame as well)